src/NHSD.BuyingCatalogue.Organisations.Api/Models/OrganisationErrors.cs

```diff
@@ -0,0 +1,12 @@
+﻿using NHSD.BuyingCatalogue.Identity.Common.Models;
+
+namespace NHSD.BuyingCatalogue.Organisations.Api.Models
+{
+    public static class OrganisationErrors
+    {
+        public static ErrorDetails OrganisationAlreadyExists()
```

Comment:
Not that important but these can both (class and method) be `internal`.

Reply:
I'm thinking about leaving them public for now. if I were to change them to be internal, I'd also need to change the ones in `ApplicationUserErrors` and I don't want to bloat this PR more than it needs to be. Hope that's understandable.

Reply:
Why would `ApplicationUserErrors` need to change? I didn't think that was part of this PR?

Are you just thinking in terms of consistency? If so, we could change `ApplicationUserErrors` later. However, as above it's not that important.

Reply:
😄 Yeah I meant for consistency sake.

So are we ok with leaving it as is for now?

Reply:
I don't mind. If we end up making any further changes I'd suggest changing it; otherwise, just leave it as is for now.

Reply:
Sounds good to me.

I'll wait and see whether there's anything else to amend after @sam-roberts-bjss re-reviews this.

If yes, I'll change it in that change set, if not, I'll leave it as is and merge.